plugins/plugin-webpack/plugin.js

```diff
@@ -158,8 +158,13 @@ module.exports = function plugin(config, args) {
   if (!cssOutputPattern.endsWith(".css")) {
     throw new Error("Output Pattern for CSS must end in .css");
   }
-  const manifest = !args.manifest ? false :
-    args.manifest === true ? './asset-manifest.json' : `${args.manifest}`;
+  
+  const manifest =
+    typeof args.manifest === string
```

Comment:
@FredKSchott latest may have broken on `string` vs `String`

Reply:
bah, thank you! my master was out of date when I released, and didn't pull in the fix for this in #1005 .

Just re-released with the fix